requirements.txt

```diff
@@ -2,3 +2,4 @@
 # Useful for dev/test jobs caches
 # Must be kept in sync with setup.py
 torch == 2.2.0 # This is what is installed in CI today
+lm-eval == 0.4.2
```

Comment:
you'll want to change this in test-requirements.txt instead of requirements.txt, it's an optional dependency